R/translate-sql-base.r

```diff
@@ -175,7 +175,42 @@ base_scalar <- sql_translator(
                         sql(ifelse(side == "both" | side == "right", "RTRIM(", "(")),
                         string
                         ,"))"
-                      )}
+                      )},
+
+  # lubridate functions
+
+  # SQL syntax reference links
+  # https://en.wikibooks.org/wiki/SQL_Dialects_Reference/Functions_and_expressions/Date_and_time_functions
+
+  # SQLite - https://www.sqlite.org/lang_datefunc.html
+  #        - https://www.sqlite.org/datatype3.html
+  #
+  as_date = sql_cast("DATE"),
+  as_datetime = sql_cast("TIMESTAMP"),
+
+  today = function(tzone = "") {build_sql("CURRENT_DATE")},
+  now = function(tzone = "") {build_sql("CURRENT_TIMESTAMP")},
+
```

Comment:
I don't think you need a curly brace for a single line funciton

Reply:
And I'd recommend omitting the `tzone` argument since you're not using it.

Reply:
Is there any value in including the tzone argument so that there is parity with the `lubridate` function itself? I.e. makes it more clear to me that there is a disparity in the way the function is handled. 

I think my main point of difficulty here is a way of matching the package function to the implementation in dbplyr. Maybe `dbtest` can move us that direction with testing (i.e. write a failing test that shows the missing functionality).

Reply:
I think it's easiest to leave it off so you get an error if it's used. If you want to keep it, you'll need to explicitly throw an error, e.g.:

```R
if (!missing(tzone)) {
  stop("`tzone` not supported in SQL translation", call. = FALSE)
}
```